README.md

```diff
@@ -108,8 +108,21 @@ You can now change the docs locally and the web page will automatically reload
 and apply your changes.
 
 ### Running the Tests
-In order to run the tests make sure that you have the development requirements installed.
+In order to run the tests, make sure that you have the development requirements installed:
+```bash
+sudo apt-get install graphviz graphviz-dev # Only on Ubuntu/Debian
+brew install graphviz wget                 # Only on macOS
+pip install -r requirements-dev.txt
```

Comment:
should we do sth like `make prepare-tests-mac` `make prepare-tests-linux` ?

Reply:
then it's also not that duplicate

Reply:
I thought of putting part of the installation instructions in a `.sh` script and have developers and Travis run it before running `make test`. I decided against it because it was getting a bit longer than I would've liked.

Putting some of the required steps on the Makefile is a good idea though. Maybe I could do the same with the following lines?
```bash
  - pip install https://github.com/explosion/spacy-models/releases/download/en_core_web_md-2.1.0/en_core_web_md-2.1.0.tar.gz#egg=en_core_web_md==2.1.0 --no-cache-dir > jnk
  - python -m spacy link en_core_web_md en
  - pip install https://github.com/explosion/spacy-models/releases/download/de_core_news_sm-2.1.0/de_core_news_sm-2.1.0.tar.gz#egg=de_core_news_sm==2.1.0 --no-cache-dir > jnk
  - python -m spacy link de_core_news_sm de
  - if [[ ! -f /tmp/cached/total_word_feature_extractor.dat ]]; then
    travis_wait wget --quiet -P /tmp/cached/ https://s3-eu-west-1.amazonaws.com/mitie/total_word_feature_extractor.dat;
    fi
  - mv /tmp/cached/total_word_feature_extractor.dat data/total_word_feature_extractor.dat
```
Because all of this are listed on .travis.yml **and** README.md, so they need to be kept in sync manually.

Reply:
I'd probably move the `travis_wait` to `travis_wait make prepare-tests`, but otherwise it looks great :-)

Reply:
Perfect. Do we need the `travis_wait`, though? The download is completing very fast, at least locally. We could also remove the --quiet flag.

Reply:
I also don't understand completely the usage of `-f` and then `mv`. If the file is being moved, then won't the `! -f` check always be true?

Reply:
I found this https://docs.travis-ci.com/user/caching/#things-not-to-cache:

> The cache’s purpose is to make installing language-specific dependencies easy and fast, so everything related to tools like Bundler, pip, Composer, npm, Gradle, Maven, is what should go into the cache.
> 
> Large files that are quick to install but slow to download do not benefit from caching, as they take as long to download from the cache as from the original source:
> 
>     Android SDKs
>     Debian packages
>     JDK packages
>     Compiled binaries
>     Docker images

Reply:
The command downloaded the file to /tmp/cached/xyz, and then moved it from there with `mv`, so then in theory the `! -f /tmp/cached/xyz` would always be true (because the file has been moved from there)